Source/WebKit/CMakeLists.txt

```diff
@@ -687,6 +687,12 @@ set(WebCore_SERIALIZATION_IN_FILES
     PlatformEvent.serialization.in
     PlatformMediaSession.serialization.in
     PlatformScreen.serialization.in
+    PlatformWheelEvent.serialization.in
+    PlaybackSessionModel.serialization.in
+    ProtectionSpaceBase.serialization.in
+    ScrollTypes.serialization.in
+    SoupNetworkProxySettings.serialization.in
```

Comment:
If its platform specific it shouldn't be in the `CMakeLists.txt`. Unfortunately there is no Soup specific CMake file in WebKit so you'll want to add these to `PlatformGTK.cmake ` and `PlatformWPE.cmake`.

You'll need to just do a `list(APPEND WebCore_SERIALIZATION_IN_FILES SoupNetworkProxySettings.serialization.in)` in each of those files.

Reply:
Moved SoupNetworkProxySettings.serialization.in as requested.